src/TextField/TextField.jsx

```diff
@@ -289,12 +279,15 @@ const TextField = React.createClass({
     ) : null;
 
     let floatingLabelTextElement = floatingLabelText ? (
-      <label
-        style={this.prepareStyles(styles.floatingLabel, this.props.floatingLabelStyle)}
+      <TextFieldLabel
+        muiTheme={this.state.muiTheme}
+        style={this.mergeStyles(styles.floatingLabel, this.props.floatingLabelStyle)}
```

Comment:
I suppose that means we would have to pass a `focused` or `isFocused` prop down to the `TextFieldLabel` so that it knows how to change styles properly. I think I'm doing something similar for the `TextFieldUnderline` component.